puppet_forge.gemspec

```
@@ -21,7 +21,7 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = '>= 1.9.3'
 
   spec.add_runtime_dependency "faraday", [">= 0.9.0", "< 0.18.0", "!= 0.13.1"]
-  spec.add_runtime_dependency "faraday_middleware", [">= 0.9.0", "< 0.14.0"]
+  spec.add_runtime_dependency "faraday_middleware", [">= 0.9.0", "<= 0.14.0"]
```

Might as well allow future patch releases of `0.14.x` as well:

```suggestion
  spec.add_runtime_dependency "faraday_middleware", [">= 0.9.0", "< 0.15.0"]
```